lib/src/collection.dart

```diff
@@ -0,0 +1,40 @@
+// ignore_for_file: avoid_print
```

Comment:
```suggestion
```

Use the all-new `logger` variable instead of printing:
```dart
import "package:burt_network/logging.dart";

logger.debug("This is to log interesting values that won't show unless the user asks for it");
logger.verbose("This is to log normal events that won't show unless the user asks for it");
logger.info("This is a regular message indicating regular stuff, and will show, so use it sparingly");
logger.warning("Something went wrong, but the code can fix it, like a try/catch");
logger.error("Something went wrong, and the code can't fix it. Probably throw an error instead of this");
```

See https://binghamtonrover.github.io/Dart-Networking/logging/logger.html